### ISSUE TITLE
[Bug] GW2UI doesn't load when using OPie

### ISSUE THREAD

**Issue by quicks1lver42** [Mortalknight/GW2_UI#221]
Not sure whether this should be reported here or to the [OPie](https://www.curseforge.com/wow/addons/opie/files/5535853) dev, but given that it's only GW2UI which breaks, I figured I'd put it here first.

### Description
GW2UI fails to show at all when also running [OPie](https://www.curseforge.com/wow/addons/opie/files/5535853). Bizarrely they worked fine together until updating GW2UI to 7.0.0 and OPie to 'zeta8a'.

### Steps to reproduce
1. Install GW2UI
2. Install OPie
3. GW2UI stops working.
4. Disabling OPie means GW2UI works again

### Other
If you have any way of me getting some logs of what's happening, I'd be happy to provide them.

**Comment by quicks1lver42** [comment]
I found the `/console scriptErrors 1` command and retrieved an Lua error. No idea if it's of any help, but here it is:

```
Message: ...face/AddOns/GW2_UI/immersive/skins/mainMenuFrame.lua:40: attempt to index field 'Right' (a nil value)
Time: Thu Jul 25 12:49:29 2024
Count: 1
Stack: ...face/AddOns/GW2_UI/immersive/skins/mainMenuFrame.lua:40: attempt to index field 'Right' (a nil value)
[string "@Interface/AddOns/GW2_UI/immersive/skins/mainMenuFrame.lua"]:40: in function <...face/AddOns/GW2_UI/immersive/skins/mainMenuFrame.lua:39>
[string "@Interface/AddOns/GW2_UI/immersive/skins/mainMenuFrame.lua"]:91: in function `SkinMainMenu'
[string "@Interface/AddOns/GW2_UI/GW2_ui.lua"]:626: in function <Interface/AddOns/GW2_UI/GW2_ui.lua:583>
[string "@Interface/AddOns/GW2_UI/GW2_ui.lua"]:873: in function <Interface/AddOns/GW2_UI/GW2_ui.lua:871>

Locals: b = Button {
}
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index field 'Right' (a nil value)"
BUTTONS = <table> {
 Options = <table> {
 }
 Edit Mode = <table> {
 }
 Return to Game = <table> {
 }
 GW2 UI = <table> {
 }
 Log Out = <table> {
 }
 MOVEANYTING = <table> {
 }
 Exit Game = <table> {
 }
 Shop = <table> {
 }
 Support = <table> {
 }
 What's New = <table> {
 }
 Rating = <table> {
 }
 AddOns = <table> {
 }
 Macros = <table> {
 }
}
GW = <table> {
 screenwidth = 1920
 animations = <table> {
 }
 oUF = <table> {
 }
 myrole = "DAMAGER"
 myname = "<snip>"
 addonName = "GW2 UI"
 myClassID = 8
 LFGPI = <table> {
 }
 mylocal = "enUS"
 INDICATORS = <table> {
 }
 BLOOD_SPARK = <table> {
 }
 myLocalizedFaction = "<snip>"
 myrace = "<snip>"
 char_equipset_SavedItems = <table> {
 }
 AURAS_IGNORED = <table> {
 }
 GridSettings = <table> {
 }
 myLocalizedRace = "<snip>"
 myclass = "<snip>"
 charSettings = <table> {
 }
 myrealm = "<snip>"
 SpacingSize = 1
 global = <table> {
 }
 MoveHudScaleableFrame = GwSmallSettingsContainer {
 }
 HiddenFrame = Frame {
 }
 socialSecure_OnAttributeChanged = "    if name ~= "windowpanelopen" then
        return
    end

    local fmFriend = self:GetFrameRef("GwFriendList")
    local showFriend = false
    local fmIgnore = self:GetFrameRef("GwIgnoreList")
    local showIgnore = false
    local fmRecruitAFriend = self:GetFrameRef("GwRecruitAFriendList")
    local showRecruitAFriend = false
    local fmWho = self:GetFrameRef("GwWhoList")
    local showWho = false
    local fmQuick = self:GetFrameRef("GwQuickList")
    local showQuick = false
    local fmRaid = self:GetFrameRef("GwRaidList")
    local showRaid = false

    local close = false
    local keytoggle = self:GetAttribute("keytoggle")

    if fmFriend ~= nil and value == "friendlist" then
        if keytoggle and fmFriend:IsVisible() then
            self:SetAttribute("keytoggle", nil)
            self:SetAttribute("windowpanelopen", nil)
            return
        else
            showFriend = true
        end
    elseif fmIgnore ~= nil and value == "ignorelist" then
    GW_CHANGELOGS = <table> {
 }
 CHANGELOGS_TYPES = <table> {
 }
 REP_TEXTURES = <table> {
 }
 LayoutManager = <table> {
 }
 QuestViewMixin = <table> {
 }
 nameRoleIconPure = <table> {
 }
 AURAS_INDICATORS = <table> {
 }
 privateDefaults = <table> {
 }
 CLASS_ICONS = <table> {
 }
 ShowRlPopup = false
 wowpatch = "11.0.0"
 Libs = <table> {
 }
 Gw2Color = ""
 mylevel = 70
 QuestInfo_Display_hooked = true
 COLOR_FRIENDLY = <table> {
 }
 myfaction = "<snip>"
 BackdropTemplates = <table> {
 }
 nameRoleIcon = <table> {
 }
 private = <table> {
 }
 UnlocalizedClasses = <table> {
 }
 InMoveHudMode = false
 AlertContainerFrame = Frame {
 }
 MOVABLE_FRAMES = <table> {
 }
 PowerBarColorCustom = <table> {
 }
 professionBagColor = <table> {
 }
 border = 1.422222
 FACTION_COLOR = <table> {
 }
 CASTINGBAR_TEXTURES = <table> {
 }
 resolution = "1920x1080"
 REALM_FLAGS = <table> {
 }
 BarAnimateTypes = <table> {
 }
 TARGET_FRAME_ART = <table> {
 }
 AlertSystem = <table> {
 }
 wowbuild = 55824
 AchievementFrameFilterDropDownDummy = Frame {
```

**Comment by Nils89** [comment]
fixed in 7.0.1
